Source/Core/Core/HW/WiimoteEmu/Camera.cpp

```diff
@@ -88,9 +88,9 @@ void CameraLogic::Update(const Common::Matrix44& transform)
       Vec3{SENSOR_BAR_LED_SEPARATION / 2, 0, 0},
   };
 
-  const auto camera_view = Matrix44::Perspective(CAMERA_FOV_Y, CAMERA_AR, 0.001f, 1000) *
-                           Matrix44::FromMatrix33(Matrix33::RotateX(float(MathUtil::TAU / 4))) *
-                           transform;
+  const auto camera_view =
+      Matrix44::Perspective(field_of_view.x, field_of_view.x / field_of_view.y, 0.001f, 1000) *
```

Comment:
I've probably missed something, but shouldn't the first argument be fov.y?

Reply:
Yeah.. probably..Thanks. :P